includes/create-theme/theme-patterns.php

```diff
@@ -160,41 +160,38 @@ public static function add_patterns_to_theme( $options = null ) {
 				$pattern        = self::prepare_pattern_for_export( $pattern, $options );
 				$pattern_exists = false;
 
-				// Check pattern is synced before adding to theme.
-				if ( 'unsynced' !== $pattern->sync_status ) {
```

Comment:
`$pattern->sync_status` is still defined in `pattern_from_wp_block` but I don't think it's used anywhere now, so we can probably remove it.